### ISSUE TITLE
Integrating with nvm or nodenv?

### ISSUE THREAD

**Issue by jcrben** [nodejs/node-chakracore#219]
Is there a reason this node-chakracore couldn't be integrated as an option in nvm or [nodenv](https://github.com/nodenv/nodenv) rather than recommending nvs?

I personally use nodenv and I really like being able to use the same language version switcher architecture across languages (pyenv, rbenv, etc). Wary of dealing with another version manager. I could probably build chakra into nodenv altho I'm not hugely motivated right now.

In the meantime I downloaded the binary and played with it directly...

**Comment by jcrben** [comment]
Also, I couldn't get time traveling debugging to work. I tried one of the v7 and one of the v8 releases.

The instructions feel a bit scattered. Could we get a very specific set of instructions? Ideally the instructions have fewer words and more code. For example a shell script which downloads a specific version of chakra and a demo folder, which contains a script to execute the proper commands, with the vscode. This is also good set up for an end-to-end test...

I encountered https://github.com/mrkmarron/RFSDemoJS but it doesn't meet this criteria. It doesn't pin a specific version of chakra, for example. Also tried following http://www.mattzeunert.com/2016/12/22/vs-code-time-travel-debugging.html but obviously it's a bit outdated at this point; the flags it uses are deprecated at this point.
